### PR TITLE
Blender 3.2.2 Add-on potential update

### PR DESCRIPTION
**Pull request by cavEpfl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
 # 'Object Alignment':
 
-Iterative closest point alignment addon for Blender
+Iterative closest point alignment addon for Blender 
+Small update from patmo's repo so that it works for Blender 3.2.2
 
 # Instructions for Use:
 
-* ...
+* Download the latest [release](https://github.com/cavEpfl/object_alignment/releases) .zip file
```
Comment (patmo141) [inline README.md]: change link to point to this repo

File: README.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
 # Instructions for Use:
 
-* ...
+* Download the latest [release](https://github.com/cavEpfl/object_alignment/releases) .zip file 
+* In Blender go Edit > Preferences > Add-ons
+* Click on Install... and select the .zip Add-on, then enable it
+
+* Manually place the two meshes close together
+* Select the base object first then the object to be aligned second and click on ICP Align
+* Increase the number of ICP iterations or just ICP Align multiple times until the tow meshes are well aligned
```
Comment (patmo141) [inline README.md]: Please add description of take_m_with

File: __init__.py [hunk 3]
```diff
@@ -18,8 +18,8 @@
 bl_info = {
     "name": "Object Alignment",
     "author": "Patrick Moore & Christopher Gearhart",
-    "version": (0, 2, 0),
```
Comment (patmo141) [inline __init__.py:21]: add your name to contributors if you like

**Review by patmo141 — CHANGES_REQUESTED** [review]
Thank you!

File: README.md (hunk 2 above)
Comment (cavEpfl) [reply README.md]: It has been added.

**Review by patmo141 — APPROVED** [review]

**Comment by patmo141** [comment]
Are you aware of the picked points method for initial alignment?  You don't have to get the initially close together if you  use that.

**Comment by cavEpfl** [comment]
Yes, but unfortunately I wasn't able to get it to work on my setup, so I preferred to skip it in the instructions and tell people to do that step manually (for the moment).

**Comment by cavEpfl** [comment]
Thank you for the merge, and don't forget to upload the latest release version like I did here : https://github.com/cavEpfl/object_alignment/releases

**Pull request merged**